pkgs/development/coq-modules/mathcomp/default.nix

```diff
@@ -19,9 +19,10 @@ let
   owner = "math-comp";
   withDoc = single && (args.withDoc or false);
   defaultVersion = with versions; lib.switch coq.coq-version [
-      { case = isGe "8.15"; out = "1.17.0"; }
+      { case = isGe "8.16"; out = "1.18.0"; }
       { case = range "8.16" "8.18"; out = "2.1.0"; }
       { case = range "8.16" "8.18"; out = "2.0.0"; }
+      { case = range "8.15" "8.18"; out = "1.17.0"; }
```

Comment:
I think that this line should be moved two lines up.

Reply:
Why not but isn't it equivalent?

Reply:
Hmm. You might be right…